src/LiveComponent/doc/index.rst

```diff
@@ -3733,6 +3733,8 @@ uses Symfony's test client to render and make requests to your components::
             ;
 
             // Submit form data
+	    // 'form' is the name of the form you're submitting (here FormType). 
+            // e.a : form submitted is EntityCreationType the name of your form would be 'entity_creation' and should replace 'form' in the below submitForm call
```

Comment:
I'd suggest to remove the text and use another word than "form" in the exemple..

Something like "create_comment"  or "my_form" .

wdyt ?

Reply:
indeed, I didn't want to update the "core" of the documentation but if it's a possibility, we could indeed do something like:

```php
// Submit form data ('my_form' for your MyFormType form)
$testComponent
    ->submitForm(['my_form' => ['input' => 'value']], 'save');
```

Reply:
I pushed that modification